packages/pigeon/lib/dart_generator.dart

```diff
@@ -49,8 +49,58 @@ String _escapeForDartSingleQuotedString(String raw) {
       .replaceAll(r"'", r"\'");
 }
 
+String _calcCodecName(Api api) => '_${api.name}Codec';
```

Comment:
What is `calc` trying to indicate here that is different from `get`? This seems like an odd name.

Reply:
I usually only use `get` when it's idiomatic for the language or when I'm retrieving a cached value.  To me `calc` means a pure function that will actually calculate something rather than look it up.  I think some of that messiness is also why some languages did away with the "get" verb for getters.

Reply:
I'd encourage you to reconsider this philosophy; I have never encountered that convention, so I have no idea what `calc` means when I read it, and I doubt very much that I'm alone in that. Using a name that means something special to you, but is somewhere between odd and actively confusing to a reader who isn't you doesn't improve readability overall. I found this actively harder to read than if it had been called `_getCodecName`, since I had to stop to figure out what `calc` was supposed to mean (and couldn't).

I'm all for using better verbs than "get" when there's a clear, more precise alternative, but avoiding "get" just to avoid it can make the code less clear.

Reply:
"Calc" is listed as an abbreviation for "calculate" at dictionary.com: https://www.dictionary.com/browse/calc

Looks like google's style guide actually sets the bar for an abbreviation if it is listed on wikipedia.  Wikipedia says for "calc" "Short for calculation, calculator, or calculus".

It doesn't seem that confusing to me and looks like a grey area in the style guide.  If it threw you off I imagine it could be even more confusing for a non-native english speaker.

Reply:
done

Reply:
> "Calc" is listed as an abbreviation for "calculate" at dictionary.com: https://www.dictionary.com/browse/calc
> 
> Looks like google's style guide actually sets the bar for an abbreviation if it is listed on wikipedia. Wikipedia says for "calc" "Short for calculation, calculator, or calculus".

It sounds like you misunderstood my concern. I understood that "calc" was short for "calculate". What I didn't understand was why it was called "calculate" rather than "get".

If someone uses an unusual verb—and this is unusual, because I have never heard someone say that they were going to "calculate" a string, especially for trivial concatenation—then it makes me assume they are trying to communicate that something unusual is happening in that method.